dev-lua/lua-openssl/lua-openssl-0.4.1.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit base eutils toolchain-funcs
```

Comment:
Well, but what other eclass handles `$PATCHES` variable? Or should I use `epatch` inside `src_prepare` instead?